plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -343,9 +380,16 @@ static void ffmpeg_source_destroy(void *data)
 static void ffmpeg_source_activate(void *data)
 {
 	struct ffmpeg_source *s = data;
+	obs_data_t *settings = obs_source_get_settings(s->source);
+
+	if (s->latest_replay_buffer) {
```

Comment:
You do not need braces for this if statement, since its contents are only a single statement and it has no "else if" or "else" branches that need branches.
https://github.com/torvalds/linux/blob/master/Documentation/process/coding-style.rst#3-placing-braces-and-spaces